server/src/controllers/jobApplicationRouter.js

```diff
@@ -18,32 +19,52 @@ jobApplicationRouter.get('/', jwtMiddleware, async (request, response) => {
       {
         model: ApplicationComment,
         as: 'applicationComments'
+      },
+      {
+        model: Attachment,
+        as: 'attachments'
       }]
   })
   response.json(jobApplications)
 })
 
 jobApplicationRouter.post('/', jobApplicationValidator, async (req, res) => {
-  const body = req.body
+  try {
```

Comment:
No need for a `try-catch` if the only thing done in `catch` is throwing an error; that would happen without the wrapping as well